### PR TITLE
Validator rewards multisign

### PR DESCRIPTION
**Pull request by Esteban100k**
With this PR, the Validator rewards can now be claimed.
The process is basically when the claim is requested, a modal appears, then it must be confirmed and the transaction will be pending the necessary signatures to be executed.
Once executed, the balance is updated and the reward is claimed from the address that requested the claim

### CODE REVIEW

File: src/helpers/wallet_helper.ts [hunk 1]
```diff
@@ -458,27 +458,34 @@ class WalletHelper {
...
+    static async buildClaimTx(
+        address: string,
+        amount: BN,
+        activeWallet: WalletType,
+        rewardOwnerAddress: string,
+        isMultisignTx: boolean
+    ) {
+        let addressRewardOwnerBuffer = ava.PChain().parseAddress(rewardOwnerAddress)
+        const claimableSigners: [number, Buffer][] = [[0, addressRewardOwnerBuffer]]
+        let rewardsOwner = new OutputOwners([addressRewardOwnerBuffer])
```
Comment (aeddaqqa) [inline src/helpers/wallet_helper.ts]: can we remove those variables if they are not used

File: src/helpers/wallet_helper.ts [hunk 2]
```diff
@@ -458,27 +458,34 @@ class WalletHelper {
...
-    static async buildClaimTx(address: string, amount: BN, activeWallet: WalletType) {
-        let addressBufferTest = ava.PChain().parseAddress(address)
-        const claimableSigners: [number, Buffer][] = [[0, addressBufferTest]]
-        let rewardsOwner = new OutputOwners([addressBufferTest])
+    static async buildClaimTx(
+        address: string,
+        amount: BN,
+        activeWallet: WalletType,
+        rewardOwnerAddress: string,
+        isMultisignTx: boolean
```
Comment (aeddaqqa) [inline src/helpers/wallet_helper.ts]: this variable is not used can we remove it, or if the check of the wallet type is needed we can check if the wallet is an instance of MultiSig Wallet

File: src/components/wallet/earn/Validate/ModalClaimReward.vue [hunk 3]
```diff
@@ -33,14 +36,21 @@
...
-                    {{
-                        $t('validator.rewards.modal_claim.confirmed_claimed', {
-                            amount: confirmedClaimedAmountText,
-                            symbol: symbol,
-                        })
-                    }}
-                </h2>
+
+                <div v-if="isMultisignTx">
+                    <h2>Tx Pending for approve</h2>
```
Comment (aeddaqqa) [inline src/components/wallet/earn/Validate/ModalClaimReward.vue]: can we use i18n translation there

File: src/components/wallet/earn/Validate/ModalClaimReward.vue [hunk 4]
```diff
@@ -90,15 +103,28 @@ export default class ModalClaimReward extends Vue {
     async confirmClaim() {
         try {
             await WalletHelper.buildClaimTx(
-                this.nodeInfo.rewardOwner.addresses[0],
+                this.pChainddress,
                 new BN(this.amount),
-                this.$store.state.activeWallet
+                this.$store.state.activeWallet,
+                this.rewardOwner,
+                this.isMultisignTx
```
Comment (aeddaqqa) [inline src/components/wallet/earn/Validate/ModalClaimReward.vue]: this variable need to be removed because its unused

**Review by Ayagoumi — DISMISSED** [review]

**Review by Ayagoumi — APPROVED** [review]

**Pull request merged**